astroquery/oac/core.py

```
@@ -0,0 +1,602 @@
+        ...                               radius=test_radius,
+        ...                               quantity="photometry",
+        ...                               attribute=["time", "magnitude",
+        ...                                          "e_magnitude", "band",
+        ...                                          "instrument"])
+
+        An example box search:
+        >>> photometry = OAC.query_region(coordinates=test_coords,
+        ...                               width=test_width, height=test_height,
+        ...                               quantity="photometry",
+        ...                               attribute=["time", "magnitude",
+        ...                                          "e_magnitude", "band",
+        ...                                          "instrument"])
+
+        These searches can be refined using the quantities, attributes, and
+        arguments, as with query_object.
+
+        """
+        # Default object name used for coordinate-based queries
+        event = 'catalog'
+
+        request_payload = self._args_to_payload(event,
+                                                quantity,
+                                                attribute,
+                                                argument,
+                                                data_format)
+
+        # Check that coordinate object is a valid astropy coordinate object
+        # Criteria/Code from ../sdss/core.py
+        if (not isinstance(coordinates, list) and
+            not isinstance(coordinates, Column) and
+            not (isinstance(coordinates, commons.CoordClasses) and
+                 not coordinates.isscalar)):
+
+            request_payload['ra'] = coordinates.ra.deg
+            request_payload['dec'] = coordinates.dec.deg
+
+        else:
+            try:
+                request_payload['ra'] = coordinates[0]
+                request_payload['dec'] = coordinates[1]
+            except Exception:
```

It would be better to try to catch only specific exception types (e.g., `IndexError`).  General exception catching can interfere with the interpreter and produce some otherwise bad results.  If the code returns an unexpected Exception type, we should just raise that Exception and deal with it as a bug